requirements-github.txt

```diff
@@ -2,10 +2,10 @@ pyyaml==6.0
 pycodestyle==2.9.1
 netCDF4==1.6.1
 matplotlib==3.5.2
-cartopy==0.21.0
+cartopy==0.21.1
 scikit-learn==1.1.2
 xarray==2022.6.0
-seaborn==0.12
+seaborn>=0.12
```

Comment:
More things that can change underneath us with non-exact versions. Can you check which version GH cloned and then set to that?

Reply:
yeah this was just a matter of throwing things at the wall and see what stuck, will update to that, test, and then make this non-draft